src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -50,7 +50,7 @@ public class CommandLineTests : CommandLineTestBase
         private static readonly string s_CSharpCompilerExecutable = Path.Combine(
             Path.GetDirectoryName(typeof(CommandLineTests).GetTypeInfo().Assembly.Location),
             Path.Combine("dependency", "csc.exe"));
-        private static readonly string s_DotnetCscRun = ExecutionConditionUtil.IsMono ? "mono" : string.Empty;
+        private static readonly string s_DotnetCscRun = ExecutionConditionUtil.IsMonoDesktop ? "mono desktop" : string.Empty;
```

Comment:
Changing `"mono"` to `"mono desktop"` looks unexpected to me. It seems that `s_DotnetCscRun` is used to execute commands. Is `mono desktop` a valid command?

Reply:
Yeh .. You are correct. Will change this.